### DISCUSSION TITLE
Showcase Your RxDBDotNet Projects!

### DISCUSSION THREAD

**Discussion by rbeauchamp** [Ziptility/RxDBDotNet#67] [Show and tell]
Welcome to our Show and Tell space! This is where you can showcase your awesome projects using RxDBDotNet. Whether you've built a small proof of concept or a full-fledged application, we want to see it!

## 🚀 Share Your Success Stories

Have you used RxDBDotNet to solve a unique problem, create an innovative application, or integrate it unexpectedly? We'd love to hear about it!

## 📸 What to Share:

- 📱 Project overview: What did you build?
- 🛠️ How RxDBDotNet helped: Which features were most helpful?
- 🏗️ Architecture insights: Any cool patterns or techniques you used?
- 📊 Performance wins: Did RxDBDotNet boost your app's performance?
- 🎨 Screenshots or demos (if possible)
- 🔗 Links to public repos or live projects (if available)

## 💡 Why Share?

- Inspire others in the community
- Get feedback and ideas for improvement
- Showcase your skills and creativity
- Help us understand how RxDBDotNet is being used in the wild

## 👀 Browsing for Inspiration?

If you're here looking for ideas:
- Leave encouraging comments
- Ask questions about implementations you find interesting
- Share how a showcased project might help with your work

Let's celebrate our successes and learn from each other! Who's ready to show off their work? 🙌

Remember: Respect intellectual property and don't share sensitive information. If you're showcasing work done for a client or employer, ensure you have permission to share.

**Comment by AndyWatkins** [comment]
Hi, I am an experienced VB.net programmer mostly building desktop edi conversions. I am looking to build a serverless Blazor Webassembly app and am trying to figure some things out. One is persistent data storage without a server. I think I really like the capabilities of RxDB and would like to explorer this, however I am currently ignorant of how the data is stored, synced, and shared. The sample project on GIT looks like it will require a server for deployment. 
My question for the Veteran C#/Blazor programmer is how can I persist data with a file stored maybe on a cloud drive and sync it. I understand RxDB may not be the best choice and maybe even Blazor is a bad choice.

thanks for any feedback.

**Reply by rbeauchamp** [reply]
I appreciate your interest in RxDBDotNet! While RxDBDotNet is a powerful library for reactive data sync in .NET applications, it's designed explicitly for scenarios with a .NET server backend using GraphQL, which doesn't align with your serverless Blazor WebAssembly requirements.

For a serverless Blazor WebAssembly application with cloud file storage sync, you might want to explore the following:
- Browser-based storage solutions like IndexedDB
- Direct integration with cloud storage services
- Blazor-specific storage libraries like Blazored.LocalStorage

Given your VB.NET background, you might also find Blazor Server a more familiar starting point for your EDI conversion application, as it allows you to use traditional .NET file I/O patterns.

Best of luck with your project!

**Comment by AndyWatkins** [comment]
Hi Richard, 

Thank you for your response. I have been mulling over some things and think The Blazor with RxDBDotNet may very well be what I need. This project isn’t for an EDI but for a custom collaboration and management application which is why RxDB stands out to me. 

I will go over your documentation and maybe try to implement the items that will make it possible for me to test your example project. This should help me understand more about how things are working and communicating between the UI/App and the backend. 

I have determined that I will need to find a relatively inexpensive service for my server needs. Any advise there would be greatly appreciated. I am trying to dive in experimentally as this has always been my best method of learning. I think I’m getting a little old, things don’t seem as clear to me right now.. haha

I really hope to have a working test with your project by the end of the weekend. I may reach out to you again for advice and guidance if it isn’t too much trouble.

 

Thanks again. 

 

Andy Watkins
